src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterPeople = century === undefined
+    ? people.filter(man => man.sex === 'm')
+    : people.filter(man => man.sex === 'm'
+    && Math.ceil(man.died / 100) === century);
+
+  const allMan = filterPeople.map(man => man.died - man.born);
+
+  const middleAge = allMan
```

Comment:
Use a more descriptive variable name instead of 'middleAge'. For example, 'averageAge'.